cis/plugins/validation/mozilliansorg_publisher_plugin.py

```diff
@@ -36,8 +36,12 @@ def run(publisher, user, profile_json):
         'groups'
     ]
 
-    # Check for any non-whitelisted attribute modification
-    # Note that no extra attribute may be added at this stage as we already performed schema validation
+    # Check the easiest case.  None type.
```

Comment:
this means we need to preimport users, until we have publishers which are allowed to create new users (workday/auth0 i believe)